yarn-error.log

```diff
@@ -0,0 +1,4654 @@
+Arguments: 
```

Comment:
You accidentally checked this in. Could you remove it and add `*.log` to `.gitignore`?